Avoid
    error: ordered comparison between pointer and zero ('const char *' and 'int')
See https://trac.macports.org/ticket/57764
--- aligner_result.cpp.orig	2016-04-21 13:26:11.000000000 -0700
+++ aligner_result.cpp	2019-01-16 09:03:47.000000000 -0700
@@ -1130,7 +1130,7 @@
 	else if(!nfilt_  ) flag = "NS";
 	else if(!scfilt_ ) flag = "SC";
 	else if(!qcfilt_ ) flag = "QC";
-	if(flag > 0) {
+	if(flag != 0) {
 		if(!first) o.append('\t');
 		o.append("YF:Z:");
 		o.append(flag);
